pkg/bootkube/create.go

```diff
@@ -44,6 +46,9 @@ func CreateAssets(manifestDir string, timeout time.Duration) error {
 	if err := wait.Poll(5*time.Second, timeout, createFn); err != nil {
 		return fmt.Errorf("Failed to create assets: %v", err)
 	}
+
+	fmt.Println("Spawning self-hosted control plane...")
```

Comment:
I won't block on this, but this just means in the future if ordering of operations changes, or new functionality is introduced between these steps -- this log message no longer makes sense. It feels weird to describe functionality in a place where that functionality doesn't occur (except by chance of ordering at this moment in time).